eng/push-tarball.sh

```diff
@@ -107,7 +107,12 @@ pushd "$vmr_path"
 
   # Re-runs should expect the branch being merged previously already
   git add -f .
-  git diff --staged --quiet || git commit -m "Update to .NET $sdk_version"
+  if git diff --staged --quiet; then
```

Comment:
I think this might be doing something else than we want.

- There's the `release/7.0` branch
- We create a new branch called something like `release/7.0-may-2023` above
- We try to open PR from the `may` branch to the `release/7.0` one

If we run this multiple times but not merge the PR, or abandon the previous PR, the new branch might be up to date already but the PR might not be there.

I think we should rather compare the new and the target branch after we commit to see if they differ and only then skip.

Reply:
Got the point, thank you :)